project/src/components/tab-item/tab-item.tsx

```diff
@@ -1,21 +1,24 @@
 import cn from 'classnames';
+import {useAppDispatch, useAppSelector} from '../../hooks/';
+import {setActiveTab} from '../../store/action';
```

Comment:
Слишком общее название
```suggestion
import {changeActiveCity} from '../../store/action';
```